napari_segment_blobs_and_things_with_membranes/__init__.py

```diff
@@ -580,7 +582,7 @@ def seeded_watershed_with_mask(membranes:ImageData, labeled_nuclei:LabelsData, m
 
 @register_function(menu="Segmentation / labeling > Seeded watershed using local minima as seeds (nsbatwm)")
 @time_slicer
-def local_minima_seeded_watershed(image:ImageData, spot_sigma: float = 10, outline_sigma: float = 0) -> LabelsData:
+def local_minima_seeded_watershed(image:ImageData, spot_sigma: float = 10, outline_sigma: float = 0) -> Tuple[LabelsData, PointsData]:
```

Comment:
Unfortunately, we cannot return tuples of types because infrastructure around this project does not support multiple return values.